air-quality-ui/src/components/summary-grid/cell/cell-rules/ColourCell.spec.ts

```diff
@@ -0,0 +1,187 @@
+//import each from 'jest-each'
```

Comment:
Can this go along with the imported package in package.json?

Reply:
I am using this module in SortingAQIComparator.spec.ts, so I am unable to remove it from package.json